recipes/google-cloud-aiplatform/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "google-cloud-aiplatform" %}
+{% set version = "1.10.0" %}
+{% set sha256 = "89d2936985eaf71a52152fe62bae9d7554dec6948032526b99ec05a049d1ee39" %}
+{% set dev_url = "https://github.com/googleapis/python-aiplatform" %}
+{% set doc_url = "https://cloud.google.com/python/docs/reference/aiplatform/latest" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
I would ask upstream to remove the empty `scripts` tag and make this noarch as that seems to be the only thing holding it up.

If that's currently not possible you can also just add a patch here removing it.

Ref: https://github.com/googleapis/python-aiplatform/blob/1c341544e9bd94c6ff0ee41177565c8c078673a3/setup.py#L114

Reply:
Thanks for this info! I've opened a PR upstream : https://github.com/googleapis/python-aiplatform/issues/1003

Reply:
Great! If you're not sure how soon a new release with that will be cut, you can also include that as a patch here. Let me know if you need any help with that.